ffv1.md

```diff
@@ -472,6 +472,52 @@ SVGI:!---
 SVGC:rangebinaryvalues7.svg=$$j_{0}=2$$
 AART:j_(0) = 2
 
+```c
+    range = 0xFF00;
+    end   = 0;
+    low   = get_bits(16);
+    if (low >= range) {
+        low = range;
+        end = 1;
+    }
+```
+Figure: A pseudo-code description of the initial states of the Range Binary Value coder context states.
```

Comment:
I prefer this description, IMO more coherent with the description in Golomb Rice mode.

Figure: A pseudo-code description of the initial states in Range Binary mode.

Reply:
Done.